File: src/annotation-web/internal/catalog/catalog.go
package catalog

import "github.com/PhysShell/relationship_fix/src/annotation-web/internal/model"

var Labels = []model.Label{
	{ID: "B.BLAME_CRITICISM", DisplayName: map[model.Language]string{model.LanguageRU: "Обвинение / критика", model.LanguageEN: "Blame / criticism"}},
	{ID: "B.PRESSURE_FOR_CHANGE", DisplayName: map[model.Language]string{model.LanguageRU: "Давление с целью изменения", model.LanguageEN: "Pressure for change"}},
	{ID: "B.VALIDATION", DisplayName: map[model.Language]string{model.LanguageRU: "Признание / понимание переживания или позиции", model.LanguageEN: "Validation / substantive acknowledgement"}},
	{ID: "B.REPAIR_ATTEMPT", DisplayName: map[model.Language]string{model.LanguageRU: "Попытка снизить накал / восстановить взаимодействие", model.LanguageEN: "Repair attempt / de-escalation"}},
	{ID: "B.AVOIDANCE_TOPIC_SHIFT", DisplayName: map[model.Language]string{model.LanguageRU: "Уход / смена поднятой темы", model.LanguageEN: "Avoidance / topic shift"}},
}

var Items = []model.Item{
	{
		ID: "dg-04", SourceLanguage: model.LanguageRU,
		Source: []model.Message{{Author: "A", Text: "Я проснулась от холода."}, {Author: "B", Text: "Ты вчера оставил окно открытым, и утром в комнате было холодно.", Target: true}},
		Presentation: map[model.Language]model.Presentation{
			model.LanguageRU: {Context: "Я проснулась от холода.", Target: "Ты вчера оставил окно открытым, и утром в комнате было холодно.", TranslationProvenance: "source"},
			model.LanguageEN: {Context: "I woke up because I was cold.", Target: "You left the window open yesterday, and the room was cold in the morning.", TranslationProvenance: "prototype_mt_v1"},
		},
	},
	{
		ID: "dg-05", SourceLanguage: model.LanguageRU,
		Source: []model.Message{{Author: "A", Text: "Мне было страшно одной ждать результаты обследования."}, {Author: "B", Text: "Понимаю, почему тебе было страшно. Это правда тяжёлое ожидание.", Target: true}},
		Presentation: map[model.Language]model.Presentation{
			model.LanguageRU: {Context: "Мне было страшно одной ждать результаты обследования.", Target: "Понимаю, почему тебе было страшно. Это правда тяжёлое ожидание.", TranslationProvenance: "source"},
			model.LanguageEN: {Context: "I was scared waiting for the test results alone.", Target: "I understand why you were scared. That really is a hard wait.", TranslationProvenance: "prototype_mt_v1"},
		},
	},
	{
		ID: "dg-06", SourceLanguage: model.LanguageRU,
		Source: []model.Message{{Author: "A", Text: "Мы опять кричим друг на друга, это уже никуда не ведёт."}, {Author: "B", Text: "Стоп. Я сейчас злой и говорю лишнее. Давай на час разойдёмся и вернёмся к этому в девять.", Target: true}},
		Presentation: map[model.Language]model.Presentation{
			model.LanguageRU: {Context: "Мы опять кричим друг на друга, это уже никуда не ведёт.", Target: "Стоп. Я сейчас злой и говорю лишнее. Давай на час разойдёмся и вернёмся к этому в девять.", TranslationProvenance: "source"},
			model.LanguageEN: {Context: "We're yelling at each other again; this isn't getting us anywhere.", Target: "Stop. I'm angry right now and saying things I shouldn't. Let's take an hour apart and come back to this at nine.", TranslationProvenance: "prototype_mt_v1"},
		},
	},
	{
		ID: "dg-07", SourceLanguage: model.LanguageRU,
		Source: []model.Message{{Author: "A", Text: "Я случайно купил билеты не на тот день 🤦"}, {Author: "B", Text: "гений планирования 😂❤️ люблю тебя, катастрофа", Target: true}},
		Presentation: map[model.Language]model.Presentation{
			model.LanguageRU: {Context: "Я случайно купил билеты не на тот день 🤦", Target: "гений планирования 😂❤️ люблю тебя, катастрофа", TranslationProvenance: "source"},
			model.LanguageEN: {Context: "I accidentally bought the tickets for the wrong day 🤦", Target: "planning genius 😂❤️ love you, disaster", TranslationProvenance: "prototype_mt_v1"},
		},
	},
	{
		ID: "dg-08", SourceLanguage: model.LanguageRU,
		Source: []model.Message{{Author: "A", Text: "Я опять всё испортил."}, {Author: "B", Text: "Ну ты гений.", Target: true}},
		Presentation: map[model.Language]model.Presentation{
			model.LanguageRU: {Context: "Я опять всё испортил.", Target: "Ну ты гений.", TranslationProvenance: "source"},
			model.LanguageEN: {Context: "I messed everything up again.", Target: "Well, you're a genius.", TranslationProvenance: "prototype_mt_v1"},
		},
	},
	{
		ID: "dg-09", SourceLanguage: model.LanguageRU,
		Source: []model.Message{{Author: "A", Text: "Я опять забыл предупредить, что задержусь."}, {Author: "B", Text: "Я не должна была на тебя орать, прости. Но мне правда нужно, чтобы ты начал писать, когда задерживаешься. Каждый раз.", Target: true}},
		Presentation: map[model.Language]model.Presentation{
			model.LanguageRU: {Context: "Я опять забыл предупредить, что задержусь.", Target: "Я не должна была на тебя орать, прости. Но мне правда нужно, чтобы ты начал писать, когда задерживаешься. Каждый раз.", TranslationProvenance: "source"},
			model.LanguageEN: {Context: "I forgot again to say I'd be late.", Target: "I shouldn't have yelled at you, I'm sorry. But I really need you to start texting me when you're running late. Every time.", TranslationProvenance: "prototype_mt_v1"},
		},
	},
}

File: src/annotation-web/src/Catalog.hs
{-# LANGUAGE OverloadedStrings #-}

-- | The dogfood catalog: annotation-ux-v7, the debug surface.
--
-- Source texts here are the RU messages of data/pilot/v0.1/form/dogfood-v7-items.yaml,
-- byte for byte; research/python/tests/test_catalog_dogfood.py checks that.
-- This is not where pilot stimuli come from: a pilot session renders its
-- sealed presentation packet and never consults this module.
--
-- EN presentations are translations and say so in their provenance string:
-- "prototype_mt_v1" for the lines that were translated for the v6 prototype,
-- "llm_translation_2026-09-08" for the lines that changed in v7. Nothing
-- scientific depends on them.
module Catalog
  ( items
  ) where

import Domain

items :: [Item]
items =
  [ Item
      { itemId = "dg-10"
      , itemSourceLanguage = RU
      , itemSource =
          [ Message "A" "Я ночью так замёрзла, кошмар." False
          , Message "B" "Ну так вентилятор всю ночь работал, ты его не выключила." True
          ]
      , itemPresentationRU = Presentation
          "Я ночью так замёрзла, кошмар."
          "Ну так вентилятор всю ночь работал, ты его не выключила."
          "source"
      , itemPresentationEN = Presentation
          "I was so cold last night, it was awful."
          "Well, the fan was running all night, you didn't turn it off."
          "llm_translation_2026-09-08"
      }
  , Item
      { itemId = "dg-05"
      , itemSourceLanguage = RU
      , itemSource =
          [ Message "A" "Мне было страшно одной ждать результаты обследования." False
          , Message "B" "Одной такое ждать — ещё бы не страшно. Там любой бы дёргался." True
          ]
      , itemPresentationRU = Presentation
          "Мне было страшно одной ждать результаты обследования."
          "Одной такое ждать — ещё бы не страшно. Там любой бы дёргался."
          "source"
      , itemPresentationEN = Presentation
          "I was scared waiting for the test results alone."
          "Waiting for something like that on your own — no wonder it was scary. Anyone would have been on edge."
          "context: prototype_mt_v1; target: llm_translation_2026-09-08"
      }
  , Item
      { itemId = "dg-06"
      , itemSourceLanguage = RU
      , itemSource =
          [ Message "A" "Мы опять кричим друг на друга, это уже никуда не ведёт." False
          , Message "B" "Стоп. Я злой сейчас, наговорю. Давай час, ну давай — я пройдусь, в девять вернусь, и нормально поговорим." True
          ]
      , itemPresentationRU = Presentation
          "Мы опять кричим друг на друга, это уже никуда не ведёт."
          "Стоп. Я злой сейчас, наговорю. Давай час, ну давай — я пройдусь, в девять вернусь, и нормально поговорим."
          "source"
      , itemPresentationEN = Presentation
          "We're yelling at each other again; this isn't getting us anywhere."
          "Stop. I'm angry right now, I'll say too much. Give me an hour, come on — I'll take a walk, be back at nine, and we'll talk properly."
          "context: prototype_mt_v1; target: llm_translation_2026-09-08"
      }
  , Item
      { itemId = "dg-07"
      , itemSourceLanguage = RU
      , itemSource =
          [ Message "A" "Я случайно купил билеты не на тот день 🤦" False
          , Message "B" "гений планирования 😂❤️ люблю тебя, катастрофа" True
          ]
      , itemPresentationRU = Presentation
          "Я случайно купил билеты не на тот день 🤦"
          "гений планирования 😂❤️ люблю тебя, катастрофа"
          "source"
      , itemPresentationEN = Presentation
          "I accidentally bought the tickets for the wrong day 🤦"
          "planning genius 😂❤️ love you, disaster"
          "prototype_mt_v1"
      }
  , Item
      { itemId = "dg-08"
      , itemSourceLanguage = RU
      , itemSource =
          [ Message "A" "Я опять всё испортил." False
          , Message "B" "Ну ты гений." True
          ]
      , itemPresentationRU = Presentation
          "Я опять всё испортил."
          "Ну ты гений."
          "source"
      , itemPresentationEN = Presentation
          "I messed everything up again."
          "Well, you're a genius."
          "prototype_mt_v1"
      }
  , Item
      { itemId = "dg-11"
      , itemSourceLanguage = RU
      , itemSource =
          [ Message "A" "Ну да, перевод забыл. Знаю." False
          , Message "B" "Ладно, орать я зря начала. Но поставь ты уже напоминалку на первое число, чтобы мне каждый месяц не писать." True
          ]
      , itemPresentationRU = Presentation
          "Ну да, перевод забыл. Знаю."
          "Ладно, орать я зря начала. Но поставь ты уже напоминалку на первое число, чтобы мне каждый месяц не писать."
          "source"
      , itemPresentationEN = Presentation
          "Yeah, I forgot the transfer. I know."
          "Okay, I shouldn't have started yelling. But set yourself a reminder for the first of the month already, so I don't have to text you every month."
          "llm_translation_2026-09-08"
      }
  ]
